lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -3026,7 +3026,7 @@ NativeCodeGenerator::GatherCodeGenData(Js::FunctionBody *const topFunctionBody,
 #endif
 
         uint objTypeSpecFldInfoCount = objTypeSpecFldInfoList->Count();
-        jitTimeData->SetGlobalObjTypeSpecFldInfoArray(RecyclerNewArray(recycler, Field(Js::ObjTypeSpecFldInfo*), objTypeSpecFldInfoCount), objTypeSpecFldInfoCount);
+        jitTimeData->SetGlobalObjTypeSpecFldInfoArray(RecyclerNewArray(recycler, Js::ObjTypeSpecFldInfo::ObjTypeSpecFldInfoPtr, objTypeSpecFldInfoCount), objTypeSpecFldInfoCount);
```

Comment:
yes, because WriteBarrierFieldTypeTraits has multiple template parameters separated by comma, the RecyclerNewArray Macro also uses comma as the separator so it will be confused 

---
In reply to: [89181575](https://github.com/Microsoft/ChakraCore/pull/2071#discussion_r89181575) [](ancestors = 89181575)